.github/workflows/PR-check.yml

```diff
@@ -0,0 +1,51 @@
+---
+name: PR Check
+
+on: 
+#  pull_request:
+#    paths-ignore:
+#      - '.github/workflows/**'
+#      - 'test-script'  
+  workflow_dispatch:
+
+env:
+  ARTIFACT_VERSION: dummy-verison 
+
+jobs:
+  build:
+    name: PR Check
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      
+      - name: Maven Cache 
+        uses: actions/cache@v3
+        with:
+          key: cache-${{ github.workflow }}
+          path: |
+            ~/.m2/repository
```

Comment:
Is there a reason to call this out separately rather than just setting `cache: maven` on the [`setup-java` action](https://github.com/actions/setup-java)?